### ISSUE TITLE
problem at TEST time

### ISSUE THREAD

**Issue by dalessioluca** [spacetx/spacetx-research#21]
there is a problem at TEST time.
It is probably related to batch normalization.
I don't like BN since it has different behavior at train and test time. 
Double check pytorch setting.....

![Screen Shot 2020-06-15 at 7 28 16 PM](https://user-images.githubusercontent.com/40577552/84722414-29663f80-af51-11ea-9a55-81ed1f36e19a.png)

**Comment by dalessioluca** [comment]
this was solved by removing the batch_norm from the UNET
